COMPATIBILITY.md

```diff
@@ -0,0 +1,25 @@
+# Compatibility
+
+We aim for the Prometheus Ruby client to be compatible with all supported
+versions of Ruby, across the MRI and JRuby platforms.
+
+Any Ruby version that has not received an End-of-Life notice (e.g.
+[this notice for Ruby 2.1](https://www.ruby-lang.org/en/news/2017/04/01/support-of-ruby-2-1-has-ended/))
+is supported.
+
+To ensure we're meeting these guidelines, we test the client against all
+supported versions, as specified in our [build matrix](.travis.yml).
+
+# Deprecation
+
+Whenever a version of Ruby fails out of support we will mirror that change in
```

Comment:
`/s/fails/falls/` ?
(I may be failing at English. If so, please ignore me)

Reply:
Yes, falls.